tests/test_proxy.cpp

```diff
@@ -48,13 +48,31 @@
 #define ID_SIZE_MAX 32
 #define QT_WORKERS    5
 #define QT_CLIENTS    3
-#define is_verbose 0
+#define is_verbose 1
```

Comment:
Macros should be all-uppercase

Reply:
it's a pre-existing one, so it's fine to avoid mixing refactor and new code additions

Reply:
You're right, I did not see that